WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -3422,3 +3435,30 @@ extension AztecPostViewController: PostEditorNavigationBarManagerDelegate {
         navigationItem.leftBarButtonItems = items
     }
 }
+
+// MARK: - Media Editing
+//
+extension AztecPostViewController {
+    func edit(_ imageAttachment: ImageAttachment) {
+        guard let image = imageAttachment.image else {
+            return
+        }
+
+        WPAnalytics.track(.mediaEditorShown)
+
+        self.mediaEditor.edit(image, from: self,
+                              onFinishEditing: { image, actions in
```

Comment:
Out of curiosity, why do you put parentheses around a single parameter in a closure but not a round more than one?

Reply:
Hmmm, do I do that?

I always try to avoid using parentheses inside closures. If I'm doing that (and I don't even remember) I'm going crazy 🤣

Reply:
😝